core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletContext.scala

```diff
@@ -34,10 +35,17 @@ import cloudflow.streamlets._
  */
 trait AkkaStreamletContext extends StreamletContext {
   private[akkastream] def sourceWithOffsetContext[T](inlet: CodecInlet[T]): scaladsl.SourceWithOffsetContext[T]
-  private[akkastream] def flowWithOffsetContext[T](outlet: CodecOutlet[T]): scaladsl.FlowWithOffsetContext[T, _]
   private[akkastream] def plainSource[T](inlet: CodecInlet[T], resetPosition: ResetPosition): Source[T, NotUsed]
   private[akkastream] def plainSink[T](outlet: CodecOutlet[T]): Sink[T, NotUsed]
 
+  private[akkastream] def committableSink[T](outlet: CodecOutlet[T], committerSettings: CommitterSettings): Sink[(T, Committable), NotUsed]
+  private[akkastream] def committableSink[T](committerSettings: CommitterSettings): Sink[(T, Committable), NotUsed]
+
+  @deprecated("Use `committableSink` instead.", "1.3.1")
+  private[akkastream] def sinkWithOffsetContext[T](outlet: CodecOutlet[T], committerSettings: CommitterSettings): Sink[(T, CommittableOffset), NotUsed]
+  @deprecated("Use `committableSink` instead.", "1.3.1")
+  private[akkastream] def sinkWithOffsetContext[T](committerSettings: CommitterSettings): Sink[(T, CommittableOffset), NotUsed]
```

Comment:
If these didn't exist before in this class and they are now deprecated, then why are they even here? 
Were they moved from somewhere else that I missed?

Reply:
They are private. It is just to mark them, these private details can be removed once we remove counterparts of these in logic. The implementation was changed.